integrations/kubernetes/kubernetes-monitoring/sensu-integration.yaml

```diff
@@ -0,0 +1,68 @@
+---
+api_version: catalog/v1
+type: Integration
+metadata:
+  namespace: kubernetes
+  name: kubernetes-monitoring
+spec:
+  class: supported # community, supported, enterprise, or partner
+  provider: monitoring
+  display_name: Kubernetes monitoring
+  short_description: Kubernetes API liveness health check.
+  supported_platforms:
+    - darwin
+    - linux
+    - windows
+  tags:
+    - kubernetes
+    - kubelet
+    - kube-apiserver
+  contributors:
+    - "@sensu"
+    - "@calebhailey"
+  prompts:
+    - type: section
+      title: Pipeline Configuration
+    - type: markdown
+      body: >-
+        Configure one or more [pipelines] for processing Kubernetes monitoring data.
+
+        [pipelines]: https://docs.sensu.io/sensu-go/latest/observability-pipeline/
+    - type: question
+      name: alert_pipeline
+      input:
+        type: string
+        title: Alert Pipeline
+        description: >-
+          How do you want to be alerted for failures detected by this pipeline (e.g. Slack or Microsoft Teams)?
```

Comment:
So in this prompt, can we make it clear that you need to type in the pipeline name? If they type one of the e.g. examples verbatim.. its not gonna end well.  The Human names for things isnt the pipeline name as understood by Sensu.

Also how do operators using the UI know what pipelines are available? They can't easily see a list of pipelines in the middle of the integration install workflow as a reference to help them make the selection can they?

Is there any way this could be a checkbox list so they can select from the pipelines in the namespace?

Reply:
@jspaleta they won't have to type at all... the `ref` field is configuration for our form generator that will result in a drop-down selector of their actually configured pipelines. 

It might not look exactly like this, but here's an example of another place in Sensu where we already support this type of "select another API resource" drop-down. 

https://user-images.githubusercontent.com/141373/155382290-81a506b9-ac63-45c5-912c-7e6165c1ae8c.mov